### ISSUE TITLE
Show an expand key for 'Copy URL' action on one blob's context menu

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#2925]
**Storage Explorer Version:** 1.13.0
**Build**:  [20200504.7](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3700661&view=results)
**Branch**: master
**Platform/OS:** Windows 10/  Linux Ubuntu 16.04/ macOS Catalina
**Architecture**: ia32/x64
**Regression From:**  Previous release(1.13.0)

**Steps to reproduce:**
1. Expand one non-ADLS Gen2 storage account -> Blob Containers.
2. Select one blob container -> Select one blob/ folder.
4. Check 'Copy URL' on its context menu.

**Expect Experience:**
No expand key shows for 'Copy URL' action.
![image](https://user-images.githubusercontent.com/41351993/81159837-004ea880-8fbc-11ea-8fe8-816aa1ba2770.png)

**Actual Experience:**
Show an expand key for 'Copy URL' action.
![image](https://user-images.githubusercontent.com/41351993/81159918-12c8e200-8fbc-11ea-9e3b-b6cdd15214b0.png)

**More Info:**
This issue also reproduces for blob snapshots context menu.

**Comment by craxal** [comment]
By design. We added functionality that allows users to copy the URL using a custom domain if a custom domain has been configured. That menu option doesn't appear if there is no custom domain configured.

@JingLouMSFT Do you think we should simplify the menu to what it was before if there is no custom domain?

**Comment by jinglouMSFT** [comment]
@craxal we certainly don't need Copy URL -> Copy URL. That is confusing.  If the sub-menu is only for copying URL with custom domain, then say so in the sub-menu.

**Comment by craxal** [comment]
@jinglouMSFT The sub menu is necessary for both Copy URL and Copy URL with Custom Domain. Here's what we can do:

If the container has no custom domain we simply have the original command:
```
Copy URL
```

If the container does have a custom domain, we have the sub menu:
```
Copy URL
├─Copy URL
└─Copy URL with Custom Domain
```

**Comment by jinglouMSFT** [comment]
Copy URL
├─Copy URL
└─Copy URL with Custom Domain
 
The structure of the menu is fine. The label for "Copy URL" sub-menu should be discussed further.

**Comment by craxal** [comment]
How about:
```
Copy URL
├─Copy URL with Default Endpoint
└─Copy URL with Custom Domain
```

**Comment by jinglouMSFT** [comment]
The sub-menus are very long.  Let's try to shorten them if we can.  Maybe just "URL with default endpoints" and "URL with custom domain" since the parent already says Copy URL?

**Comment by craxal** [comment]
```
Copy URL
├─With Default Endpoint
└─With Custom Domain
```

**Comment by v-xianya** [comment]
This issue is fixed on build [20200521.3](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3746638&view=results). So we close it.
